### ISSUE TITLE
Greenfield first run fails under OSX 10.12.1

### ISSUE THREAD

**Issue by bxlewi1** [coryhouse/react-slingshot#335]
Node version: 7.2

npm version: 3.10.9

Operating system: 10.12.1

Command line used: npm start -s

Steps to reproduce:
Download bundle from github
Extract to directory
Run npm install
Run npm -start -s

App complains about the 'import' statement.

Fails complaining about a missing lint config file if changing out npm install with npm run setup.

**Comment by bxlewi1** [comment]
This appears to only be an issue when downloading the zip.

**Comment by kwelch** [comment]
I would ensure that all dot files (`.eslintrc`, `.babelrc`, etc) were downloaded and copied properly.  The OS does not by default typically copy these files.

**Comment by kwelch** [comment]
Did copying the hidden dot files fix this issue? Is it good to close?

**Comment by nickytonline** [comment]
@coryhouse, I'm guessing you read Dan Abramov's recent tweet? https://mobile.twitter.com/dan_abramov/status/830776755204063232

**Comment by coryhouse** [comment]
Yes, though I've been aware for awhile. The most common issue on my courses is people misplacing/miscopying the dotted files. If I could turn back time on my previous recordings... :)

**Comment by kwelch** [comment]
I saw it as well and thought about making the change, but it could make the package.json overwhelming. It is currently 100+, if we put all the dot files in it would be difficult to discover [see tweet](https://twitter.com/thejameskyle/status/830901906914496513)

**Comment by coryhouse** [comment]
That's a valid concern, especially since dot files are the far more common approach. It's the first place I (and I suspect most people) look.

**Comment by kwelch** [comment]
I believe it is more valid and approachable for this type of project to leverage the `package.json` file. There have been a few times I have noticed issues created just in slingshot due to the hidden attribute of the dot files.
